api/envoy/extensions/filters/network/rbac/v3/rbac.proto

```diff
@@ -87,4 +89,10 @@ message RBAC {
   // every payload (e.g., Mongo, MySQL, Kafka) set the enforcement type to
   // CONTINUOUS to enforce RBAC policies on every message boundary.
   EnforcementType enforcement_type = 4;
+
+  // Delay the specified duration before closing the connection when the policy evaluation
+  // result is DENY. If this is not present, the connection will be closed immediately.
+  // This is useful to provide a better protection for Envoy against clients that retries
+  // aggressively when the connection is rejected by the RBAC filter.
+  google.protobuf.Duration delay_deny = 8;
```

Comment:
Thanks for the contribution. LGTM to the API overall. Only one quesion: will we expect a minimum value if this is set explicitly?

Reply:
/wait-any

Reply:
I imagine
missing means close immediately.
0 means close ASAP but effectvely to next dispatcher cycle
> 0 means an larger delay

Reply:
I think we should not impose a minimum value on this, it's an optional field and 0 or unset means closing the TCP connection immediately, which is the exact same behavior as today.

Reply:
Pratically a very small delay should be equivalent to next dispatch cycle.
If so, I guess a min deley is not reasonable. 

@wbpcode  WDYT if keep the current impl:
0 or field not present: close immediately, back compatiblity
> 0 value: the intended delay

Reply:
hmmmm, Okay.